spring-rabbit/src/test/java/org/springframework/amqp/rabbit/annotation/EnableRabbitTests.java

```diff
@@ -188,9 +216,9 @@ public Listener listener() {
 		static class Listener {
 
 			@RabbitListener(bindings =
-					@QueueBinding(value = @Queue(value = "foo", ignoreDeclarationExceptions = "true"),
-								exchange = @Exchange(value = "bar", ignoreDeclarationExceptions = "true"),
-								key = "baz", ignoreDeclarationExceptions = "true"))
+			@QueueBinding(value = @Queue(value = "foo", ignoreDeclarationExceptions = "true"),
+					exchange = @Exchange(value = "bar", ignoreDeclarationExceptions = "true"),
+					key = "baz", ignoreDeclarationExceptions = "true"))
 			public void handle(String foo) {
```

Comment:
Yeah... Please, revert on the merge.
I really like that style, but can't reach it with IDEA setting. 😢